index.html

```diff
@@ -1,74 +1,173 @@
+        <p>
+          Traveling expedition modern, clean webdesign blogger clean website
+          theme website modern. Design pretty design, travelblogger adventure
+          WordPress wanderlust theme blogger website expedition theme
+          travelblogger. Adventure fun traveler preety design website
+          expedition. Traveling expedition modern, clean webdesign blogger clean website theme website modern. Design pretty design, travelblogger adventure WordPress wanderlust theme blogger website expedition theme travelblogger. Adventure fun traveler preety design website expedition. 
+          <p>  Traveling expedition modern, clean webdesign blogger clean website
+            theme website modern. Design pretty design, travelblogger adventure
+            WordPress wanderlust theme blogger website expedition theme
+            travelblogger. Adventure fun traveler preety design website
+            expedition. Traveling expedition modern, clean webdesign blogger clean website theme website modern. Design pretty design, travelblogger adventure WordPress wanderlust theme blogger website expedition theme travelblogger. Adventure fun traveler preety design website expedition. </p>
+        </p>
       </div>
-      <div class="destination">
-        <h4>Mountain Excursion</h4>
-        <p>Expedition excursion design excursion fun, clean simple organized WordPress Travel colorful webdesign. Traveler blogger website design expedition clean excursion traveling.</p>
-        <div class="btn">Sign Me Up!</div>
       </div>
-      <div class="destination">
-        <h4>Island Getaway</h4>
-        <p>Expedition excursion design excursion fun, clean simple organized WordPress Travel colorful webdesign. Traveler blogger website design expedition clean excursion traveling.</p>
-        <div class="btn">Sign Me Up!</div>
-      </div>
-    </section>
-  </div><!-- container -->
-
-  <footer class="footer">
-    <p>Copyright Fun Bus 2018</p>
-  </footer>
+      <section class="content-section">
+        <div class="text-content">
+          <h2>Let's Go!</h2>
+          <p>
+            Adventure webdesign pretty design design, excursion cute WordPress
+            blogger design webdesign adventure. Pretty simple traveling fun
+            WordPress wanderlust darn simple organized.
+          </p>
+          <p>
+            Expedition colorful design simple excursion blogger blogger design
+            WordPress design, design organized website theme.
+          </p>
+        
+        </div>
+      
+        <div class="img-content">
+            <div
+            data-aos="slide-left"
+            data-aos-offset="200"
+            data-aos-delay="50"
+            data-aos-duration="1000"
+            data-aos-easing="ease-in-out"
+            data-aos-mirror="true"
+            data-aos-once="false">
+            <img src="img/adventure.jpg" alt="Let's go on an adventure!" />
+            </div>
+        </div>
```

Comment:
It looks like the animation is causing the screen to have a horizontal scroll. You could get rid of the scroll by setting `overflow-x: hidden` on your main container or body tag.